otj-kafka/src/test/java/com/opentable/kafka/session/BaseSessionTest.java

```diff
@@ -63,7 +63,7 @@ public void performTestSimpleWithStategy() throws ExecutionException, Interrupte
 
         // Has consumed a total of N messages
         assertThat(eventTypes.stream().filter(t -> t == ConsumedEvent.EventType.POLL).count()).isGreaterThanOrEqualTo(totalMessages);
-        moreAssertionsForTestResult(withSleep, simpleTestResult, (long) totalMessages, withSleep ? 1L : 0L);
+        moreAssertionsForTestResult(withSleep, simpleTestResult, (long) totalMessages, withSleep ? 2L : 1L);
```

Comment:
Ok, so here I really need a clear explanation of why these changed, and why these fixes were needed.